src/xcode/ENA/ENAUITests/ENAUITests.swift

```diff
@@ -128,7 +128,7 @@ class ENAUITests: XCTestCase {
 		// ScreenShot_0006: Negative result
 		try navigateThroughOnboarding()
 		XCTAssertTrue(app.buttons[AccessibilityIdentifiers.Home.submitCardButton].waitForExistence(timeout: .medium))
-		app.buttons[AccessibilityIdentifiers.Home.submitCardButton].tap()
```

Comment:
I see stuff like this all over the place. Why do we have to explicitly select the first one here? 🤯

Reply:
TBH I just wanted to fix the Screenshot on this part, I will talk with @nickguendling that we improve here, since he is working on the new UITests for the RAT home. My guess would be that we then have better identifiers :)